### DISCUSSION TITLE
Draft Issue's description won't auto-complete GH usernames, but does link.

### DISCUSSION THREAD

**Discussion by Relequestual** [community/community#58043] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [ ] Issues (The feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).
- [X] Projects (For planning and tracking work on GitHub - not any open source or other type of project).

### Body

Hello. First of all, really love Projects. Super useful.

I noticed that when you're editing the description of a "draft Issue", you don't get autocomplete for GitHub usernames, like elsewhere. When you're done editing, the saved and displayed content links to the GitHub username.

Although this isn't a showstopper, it would be emarissing if someone typoed a username and the intended target never found out.

Given that when viewing the description, the usernames are hyperlinked, it feels like a bug.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Sandro642** [comment]
Hi, @Relequestual 

Thank you for your feedback regarding autocomplete for GitHub usernames when editing the description of a "draft Issue" in Projects. While I don't have access to specific development plans, I can offer a possible solution for your concern.

To ensure consistency and avoid potential typos, you can manually type the "@" symbol followed by the GitHub username in the description field. This way, even if autocomplete is not available during editing, the intended target will still be notified when the issue is saved and displayed.

Although it may feel like a bug or missing feature that autocomplete is not available for usernames in the description of a draft issue, manually typing the usernames can be a reliable workaround to ensure proper linking and notification.

You can also consider providing feedback directly to GitHub through their official channels, such as their community forums or the "Give feedback" option available on GitHub's website. This allows the GitHub team to consider user suggestions and potentially implement improvements in future updates.

Please answered my responses if is good ;)
Sandro642

**Reply by Relequestual** [reply]
Hey @Sandro642. Thanks for replying. Right, this is exactly what I did.
I thought this was providing feedback directly to GitHub through official channels?
If it is not, the Issue template might need revising to reflect this.

**Reply by Sandro642** [reply]
I agree with you
